lib/csa/ccm/matrix.rb

```diff
@@ -193,11 +207,52 @@ def self.from_xlsx(version, input_file)
     matrix
   end
 
+  def self.fill_answers(answers_yaml_path, template_xslt_path, output_xslt_path)
+    answers = YAML.load(File.read(answers_yaml_path, encoding: 'UTF-8'))['ccm']['answers']
+    answers_hash = Hash[*answers.collect { |a| [a['question-id'], a] }.flatten]
+
+    matrix = Matrix.new(
```

Comment:
Also need to ensure that the Answers YAML uses the same version as the Template.